src/Avalonia.Themes.Fluent/TextBox.xaml

```diff
@@ -28,6 +33,7 @@
     <Setter Property="MinWidth" Value="{DynamicResource TextControlThemeMinWidth}" />
     <Setter Property="Padding" Value="{DynamicResource TextControlThemePadding}" />
     <Setter Property="FocusAdorner" Value="{x:Null}" />
+    <Setter Property="ContextMenu" Value="{DynamicResource DefaultTextBoxContextMenu}" />
```

Comment:
Why we can't use StaticResource here? I don't think "DefaultTextBoxContextMenu" resources is supposed to be changed in runtime.